nasa-gibs/worldview
# WV-2852 Tour Story Transition Duration Adjustment

## Description
This changes the duration of tour story transitions to better fit some transitions. The change factors in the startZoom and endZoom as both an independent variable and as a modifier to the distance, as well as adding a upper cap to duration.

## How To Test
1. `git checkout wv-2852-tourstory-transition`
2. `npm ci`
3. `npm run watch`
4. Open any Tour Story.
5. Proceed through each transition and verify that the speed and duration of the transition is correct.
